boards/arc/nsim/board.cmake

```diff
@@ -2,29 +2,18 @@
 set(EMU_PLATFORM nsim)
 
 if(NOT (CONFIG_SOC_NSIM_HS_SMP OR CONFIG_SOC_NSIM_HS6X_SMP))
-board_set_flasher_ifnset(arc-nsim)
+  board_set_flasher_ifnset(arc-nsim)
 board_set_debugger_ifnset(arc-nsim)
+
+board_runner_args(arc-nsim "--props=${BOARD}.props")
+set(NSIM_PROPS "${BOARD}.props")
```

Comment:
Just a nitpick: please fix formatting here (indentation) for the whole if block.